src/GtirbZeroBuilder.cpp

```diff
@@ -114,13 +114,16 @@ gtirb::from_iterator gtirb::auxdata_traits<ElfSymbolInfo>::fromBytes(ElfSymbolIn
 void buildSections(gtirb::Module &module, std::shared_ptr<BinaryReader> binary,
                    gtirb::Context &context)
 {
+    std::map<gtirb::UUID, SectionProperties> dwarfSections;
+    std::map<gtirb::UUID, SectionProperties> allSections;
```

Comment:
This doesn't seem to be updated - should this be a union of `dwarfSections` and `sectionProperties`?